README.md

```diff
@@ -336,6 +336,29 @@ function onInputKeyDown(event) {
 />
 ```
 
+### Supporting browser autofill
+
+Add the `autoComplete` prop to your select. Please notice that this feature is ignored for a Multiselect. The `autosize` prop should be set to `false` because autosize will shrink the input so the clickable area to trigger autofill suggestions does not expand over the whole select field.
```

Comment:
Let's link to https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocomplete for a list of values.

```
The value should be from [this list of possible values](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input#attr-autocomplete).
```

Reply:
Could also give some examples where a select is commonly used, like 'cc-exp-month', 'cc-exp-year'